warehouse/config.py

```diff
@@ -525,7 +525,7 @@ def configure(settings=None):
         "warehouse.account.accounts_search_ratelimit_string",
         "ACCOUNTS_SEARCH_RATELIMIT_STRING",
         default="100 per hour",
-    ),
+    )
```

Comment:
I _think_ not -- I tried this in the REPL, and it looks like it results in the return being a tuple, but the config itself is produced as a side effect of `maybe_set` rather than the return value.

For example:

```python
>>> def foo():
...     ...
... 
>>> foo(
...     
... ),
(None,)
```

(TIL this is legal at all -- I know about trailing commas for tuple creation during assignment, but I didn't know it could also be done after a bare function call!)

Reply:
Right right - `maybe_set` is mutating the passed in `settings`, and does nothing with the return value.
So not an issue here, but definitely something to keep in mind - since a stray comma can bork an expected response:

```python
>>> def foo():
...    ...
...
>>> r1 = foo()
>>> r2 = foo(),
>>> r1 == r2
False
```